pkg/storaged/vdo-monitor.py

```diff
@@ -97,6 +97,11 @@ def monitor_volume(dev):
                          'usedPercent', 'savingPercent'
     ]
 
+    try:
+        dev = Samples.samplingDevice(dev, dev)
+    except:
+        pass
```

Comment:
This could use a comment. Does this have any side effects that are being checked elsewhere? The result isn't being used anywhere, there is no warning etc.?

Reply:
Yes, this is too sloppy.  I should at least use `expect AttributeError`.